assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -21,109 +21,119 @@
 	border-radius: 4px;
 	color: $c-text-notice;
 	margin: 1.5em 0;
-	padding: $grid-gap-phone;
 
-	@media (min-width: $bp-desktop) {
-		padding: $grid-gap-desktop;
-	}
 
 	&--warning,
 	&--info,
 	&--suggestion {
 
-		.googlesitekit-settings-notice__text {
-			padding-left: 30px;
-			position: relative;
-
-			&::before {
-				background-position: center center;
-				background-repeat: no-repeat;
-				background-size: 2px 12px;
-				border-radius: 50%;
-				border-style: solid;
-				border-width: 2px;
-				content: "";
-				display: block;
-				height: 22px;
-				left: 0;
-				position: absolute;
-				width: 22px;
-			}
+		svg {
+			height: 20px;
+			width: 20px;
 		}
-	}
 
-	&--warning {
-		background-color: $c-background-notice-warning;
-		color: $c-text-notice-warning;
+		.googlesitekit-settings-notice__row {
```

Comment:
Styles still don't look right to me.

* Learn more is still shown in the middle:
  ![google github io_site-kit-wp_storybook_pull_3404__path=_story_modules-analytics-setup-setupformga4transitional--without-existing-tag](https://user-images.githubusercontent.com/873049/121184959-44227a00-c86e-11eb-8e1a-f8f4793fc9bd.png)
  https://google.github.io/site-kit-wp/storybook/pull/3404/?path=/story/modules-analytics-setup-setupformga4transitional--without-existing-tag
* Learn more should be displayed bellow the notice text on mobile devices:
  ![google github io_site-kit-wp_storybook_pull_3404__path=_story_modules-analytics-setup-setupformga4transitional--without-existing-tag (1)](https://user-images.githubusercontent.com/873049/121185313-98c5f500-c86e-11eb-8536-4bfeeadce46d.png)
  https://google.github.io/site-kit-wp/storybook/pull/3404/?path=/story/global-notices--settings-info-notice-multi-line

Reply:
I was about to ask about the mobile view as there's no designs

Should it be right or left aligned?  I assume the star should stay on the same row (current on `main` it's all sort of on one-line)

![another idea](https://user-images.githubusercontent.com/8496063/121188490-741b4e80-c869-11eb-8438-7a9bc1815827.png)
![idea-1](https://user-images.githubusercontent.com/8496063/121188494-74b3e500-c869-11eb-8fa7-b4205eb8af68.png)

Reply:
I'm gonna make the executive decision of left aligned :joy: 

Then I'll assign the ticket back

Reply:
> Should it be right or left aligned? I assume the star should stay on the same row (current on main it's all sort of on one-line)

I think it should be something like this: 
![google github io_site-kit-wp_storybook_pull_3404__path=_story_modules-analytics-setup-setupformga4--without-existing-tag](https://user-images.githubusercontent.com/873049/121197882-5d312800-c87a-11eb-83ec-1c98398cc598.png)

Reply:
Right yes, I see that now.  Small fix but I'll do that now

Reply:
Done :+1: